README.md

```diff
@@ -102,6 +102,107 @@ Withdrawal flow:
    withdrawal that was created in the previous step. This will finalize the
    LockedGold withdrawal and return the remaining CELO to the user.
 
+
+## Updating Devchain Chain Data with Staked CELO Contracts
+
+This section will walk you through how to generate a new devchain tarball containing deployed staked CELO contracts.
+
+### Deploying Staked CELO Contracts to Local Devchain
+
+First open a terminal and run the `devchain` and specify the path to store the chain data. This will need to stay running until we have generated the new tarball.
+
+```bash
+yarn devchain --db db/
+```
+
+In a separate terminal, deploy all staked CELO contracts to the current local network.
+
+```bash
+yarn deploy:devchain
+```
+
+This will create `deployments/local/` directory that contains functions to easily access deployments using hardhat-deploy. 
+
+See [hardhat-deploy](https://github.com/wighawag/hardhat-deploy#migrating-existing-deployment-to-hardhat-deploy) for more details on how to use existing deployments.
+
+Once deployed, devchain can safely be stopped.
+
+### Generating New Chain Data Tar
+
+Once the contracts are successfully deployed to the network, compress the chain data.
+
+<!-- TODO(soloseng): ∆ checkout branch on PR merge -->
+
+```bash
+# Start with a fresh checkout to avoid build complications.
+> git clone https://github.com/celo-org/celo-monorepo.git
+> git fetch --all --tags
+> git checkout soloseng/add-devchain-option
```

Comment:
so this is technically checking out a new branch right ..i.e checkout -b name-of-new-branch

Reply:
Yes, it's currently checking out a branch that gives us the option to compress the chaindata. (this is temp until branch is merged with `master`.

@m-chrzan thinking more about it, I'm wondering if it might be a bit overkill to clone the `CELO-MONOREPO` only to compress some files vs a script in `STAKED-CELO` repo to handle the compression. Thoughts?

Reply:
What do you think about having a simple wrapper script here around the monorepo devchain script that takes a `--monorepo` flag (which assumes the monorepo is already built and checked out at the correct branch)?

Reply:
Added a script that assumes monorepo is available locally. Details on how to use in the README.